reactor-core/src/main/java/reactor/core/publisher/FluxProcessor.java

```diff
@@ -202,7 +201,9 @@ public Object scanUnsafe(Attr key) {
 	public final FluxSink<IN> sink(FluxSink.OverflowStrategy strategy) {
```

Comment:
maybe hint at the limitation to IGNORE if buffer is unlimited, cite `TopicProcessor` as an example and say that using `onBackpressureXXX` on the processor afterward could still make sense?